contribute-to-tidb/community-guideline.md

```diff
@@ -0,0 +1,27 @@
+# TiDB Community Guideline
+
+TiDB governs by meritocracy and adopts a model consists of reviewer, committer, and maintainer. We believe that it is important to create an inclusive community where everyone can use, contribute to, and influence the direction of the project. See [the contributors page](https://github.com/apache/tvm/graphs/contributors) for the current list of contributors.
+
+## General Development Process
+
+Everyone in the community is welcomed to send patches, documents, and propose new directions to the project. The key guideline here is to enable everyone in the community to get involved and participate the decision and development. When major changes are proposed, a design document should be sent to allow discussion by the community. We encourage public discussion, archivable channels such as GitHub issues and the [TiDB Internals forum](https://internals.tidb.io/), so that everyone in the community can participate and review the process later.
+
+Code reviews are one of the key ways to ensure the quality of the code. High-quality code reviews prevent technical debt for long-term and are crucial to the success of the project. A pull request needs to be reviewed before it gets merged. A committer who has the expertise of the corresponding area would moderate the pull request and the merge the code when it is ready. The corresponding committer could request multiple reviewers who are familiar with the area of the code. We encourage contributors to request code reviews themselves and help review each other's code – remember everyone is volunteering their time to the community, high-quality code review itself costs as much as the actual code contribution, you could get your code quickly reviewed if you do others the same favor.
+
+The community should strive to reach a consensus on technical decisions through discussion. We expect committers and maintainers to moderate technical discussions in a diplomatic way, and provide suggestions with clear technical reasoning when necessary.
+
+## Committers
```

Comment:
should we introduce the reviewer firstly?

Reply:
Yes. Updated.